### PR TITLE
Make jaeger optional for local

### PR DESCRIPTION
**Pull request by abbi-gaurav**
**Related issue(s)**
See also #983

### CODE REVIEW

**Comment by marcobebway** [comment]
Fixed some issues in jaeger helm charts:
* remove duplicate `spec.type` for `jaeger-collector` service.
* add missing `global.istio.gateway.name` entry in `jaeger/values.yaml`.
* removed the `core-` prefix from jaeger virtualservice.

File: installation/resources/installer-cr-cluster.yaml.tpl [hunk 1]
```diff
@@ -29,3 +29,5 @@ spec:
       namespace: "heptio-ark"
     - name: "logging"
       namespace: "kyma-system"
+    - name: "jaeger"
+      namespace: "kyma-system"
```
Comment (abbi-gaurav) [inline installation/resources/installer-cr-cluster.yaml.tpl]: Please add a newline

Comment (marcobebway) [reply installation/resources/installer-cr-cluster.yaml.tpl]: Done.

**Comment by marcobebway** [comment]
@abbi-gaurav I added a section describing how to install Jaeger locally on a running Kyma instance. Please check.

File: resources/jaeger/README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Jaeger
+
+## Overview
+[Jaeger](http://jaeger.readthedocs.io/en/latest/) is a monitoring and tracing tool for microservices-based distributed systems.
+
+## Details
+The Envoy sidecar uses Jaeger to trace the request flow in the Istio service mesh. The communication from Istio and Envoy uses the Zipkin protocol. Jaeger provides compatibility with the Zipkin protocol. This allows you to use Zipkin protocol and clients in Istio, Envoy, and the Kyma services.
```
Comment (bszwarc) [inline resources/jaeger/README.md]: Can we have here that:
`Istio and Envoy use the Zipkin protocol to communicate with the tracing back-end.` ?

File: resources/jaeger/README.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+For more details, see the [Istio Distributed Tracing](https://istio.io/docs/tasks/telemetry/distributed-tracing.html) documentation.
+
+## Installing Jaeger locally
+Jaeger installation is optional and by default it is not installed locally. However, it is possible to install it on a Kyma instance running locally using Helm.
```
Comment (bszwarc) [inline resources/jaeger/README.md]:
```
Jaeger installation is optional, and you cannot install it locally by default. However, you can install it on a Kyma instance and run it locally using Helm.

To install Jaeger, run the following command:
```

File: resources/jaeger/README.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Jaeger
+
+## Overview
+[Jaeger](http://jaeger.readthedocs.io/en/latest/) is a monitoring and tracing tool for microservices-based distributed systems.
```
[the remark this answers is not in the document]
Comment (marcobebway) [reply resources/jaeger/README.md]: Done.

File: resources/jaeger/README.md (hunk 2 above)
Comment (marcobebway) [reply resources/jaeger/README.md]: Done.

File: resources/jaeger/README.md (hunk 3 above)
Comment (marcobebway) [reply resources/jaeger/README.md]: Done.

File: resources/jaeger/README.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+## Details
+The Envoy sidecar uses Jaeger to trace the request flow in the Istio service mesh. Istio and Envoy use the Zipkin protocol to communicate with the tracing back-end. Jaeger provides compatibility with the Zipkin protocol. This allows you to use Zipkin protocol and clients in Istio, Envoy, and the Kyma services.
+
+For more details, see the [Istio Distributed Tracing](https://istio.io/docs/tasks/telemetry/distributed-tracing.html) documentation.
+
+## Installing Jaeger locally
+Jaeger installation is optional, and you cannot install it locally by default. However, you can install it on a Kyma instance and run it locally using Helm.
+
+To install Jaeger, run the following command:
+```bash
```
Comment (abbi-gaurav) [inline resources/jaeger/README.md]: Please mention the directory from which we should run this command

Comment (marcobebway) [reply resources/jaeger/README.md]: Done.

File: resources/jaeger/README.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+
+## Details
+The Envoy sidecar uses Jaeger to trace the request flow in the Istio service mesh. Istio and Envoy use the Zipkin protocol to communicate with the tracing back-end. Jaeger provides compatibility with the Zipkin protocol. This allows you to use Zipkin protocol and clients in Istio, Envoy, and the Kyma services.
+
+For more details, see the [Istio Distributed Tracing](https://istio.io/docs/tasks/telemetry/distributed-tracing.html) documentation.
+
+## Installing Jaeger locally
+Jaeger installation is optional, and you cannot install it locally by default. However, you can install it on a Kyma instance and run it locally using Helm.
+
+To install Jaeger, navigate to the directory `~/go/src/github.com/kyma-project/kyma/resources/` and run the following command:
```
Comment (bszwarc) [inline resources/jaeger/README.md]: Minor comment:

```
To install Jaeger, go to the `~/go/src/github.com/kyma-project/kyma/resources/` directory and run the following command:
```

**Comment by abbi-gaurav** [comment]
@derberg created a follow-up https://github.com/kyma-project/kyma/issues/1780 based on your comments.

**Pull request merged**